conda/environment.yml

```diff
@@ -10,105 +10,151 @@ channels:
   - conda-forge
 
 dependencies:
-  - python==3.10.8
+  - python~=3.11.0
   # Required until https://github.com/jupyterhub/repo2docker/pull/1196 is merged
-  - jupyterhub-singleuser>=3.0,<4.0
+  - jupyterhub-singleuser~=4.0.1
   # nbgitpuller is very helpful when distributing user content
-  - nbgitpuller==1.1.1
+  - nbgitpuller~=1.2.0
   # Specify major version of jupyterlab to use. Manually bump it when you want to upgrade
   # jupyterlab version
-  - jupyterlab>=3.0,<4.0
+  - jupyterlab>=4.0,<4.1
   # This provides memory usage stats on JupyterLab and classic notebook
-  - jupyter-resource-usage==0.6.4
+  - jupyter-resource-usage~=1.0.1
   # https://syncthing.net/ provides a dropbox-like interface for syncing directories across users,
-  # hubs and machines. Used along with `jupyter-syncthing-proxy` from pip.
-  - syncthing==1.22.1
+  # hubs and machines. Used along with `jupyter-syncthing-proxy`.
+  - jupyter-syncthing-proxy==1.0.3
+  - syncthing~=1.22.1
   # gh-scoped-creds allows users to securely push to GitHub from their repo.
   # https://blog.jupyter.org/securely-pushing-to-github-from-a-jupyterhub-3ee42dfdc54f has
   # some more information
   - gh-scoped-creds==4.1
-  # needed for Linux desktop to work
-  - websockify>=0.10
-  # Cloud access tools and libraries
-  # Access S3 easily from python
-  - s3fs==2022.11.0
-  # handle login and similar details for accessing earthdata protected data
-  # https://github.com/nsidc/earthaccess
-  - earthaccess~=0.8.2
-  # Access datasets exposed via STAC
-  - pystac-client==0.5.1
-  # Access datasets exposed via intake catalogs
-  - intake==0.6.6
-  - intake-esm==2022.9.18
-  - intake-xarray==0.6.1
+  # More git tools from github-cli
+  - gh~=2.32.1
+ 
+  # Jupyter tools
+  - ipython~=8.20.0
+  - ipywidgets~=8.1.1
+  - jupyter-book~=1.0.0
+  - jupyterlab-favorites~=3.2.1
+  - jupyterlab-geojson~=3.4.0
+  - jupyterlab-git~=0.50.0
+  - jupyterlab-h5web~=11.1.0
+  - jupyterlab-myst~=2.4.2
+  - jupyterlab_pygments~=0.3.0 # To bring extension uptodate
+  - jupytext~=1.16.1
+  - nbconvert==6.5.3
```

Comment:
Maybe remove this line, following https://github.com/CryoInTheCloud/hub-image/pull/117? (that PR still needs to be approved btw).

```suggestion
```

Reply:
Great, thank you for the catch and the reminder to review that PR!